.coveragerc_nogpu

```diff
@@ -0,0 +1,6 @@
+[report]
+# omit files that require a GPU
+omit =
+    lasagne/layers/corrmm.py
```

Comment:
When trying this locally with pytest (and `--cov-config`), I also needed to omit `lasagne/tests/*`, otherwise it would report the coverage of those.  See `.coveragerc` in the repository.